src/components/MultiSelect/helpers.ts

```diff
@@ -0,0 +1,15 @@
+import { MultiSelectSize } from './MultiSelect';
+
+export const getSizeClasses = (size: MultiSelectSize, hasActiveKeys: boolean) => {
```

Comment:
I don't think we should return an empty string and having Medium size as the default anyway, I think that part won't even be reached.

We could simply have it like this:
`export const getSizeClasses = (size: MultiSelectSize, hasActiveKeys: boolean) => {
    if (size === MultiSelectSize.Small) {
        if (hasActiveKeys) {
            return 'tw-py-1';
        } else {
            return 'tw-py-2';
        }
    }
    return 'tw-py-[11px]';
};`

Don't you think?

Reply:
Sure, we can do that 😇